modules/http/src/main/java/org/mule/extension/http/internal/request/HttpRequestOperations.java

```diff
@@ -87,19 +85,19 @@ public void request(@Optional(defaultValue = "/") String path,
       HttpRequesterRequestBuilder resolvedBuilder = requestBuilder != null ? requestBuilder : new HttpRequesterRequestBuilder();
       UriParameters uriParameters = client.getDefaultUriParameters();
 
-      String resolvedHost = resolveIfNecessary(overrides.getHost(), uriParameters.getHost(), muleEvent);
-      Integer resolvedPort = resolveIfNecessary(overrides.getPort(), uriParameters.getPort(), muleEvent);
-      String resolvedBasePath = config.getBasePath().apply(muleEvent);
+      String resolvedHost = resolveIfNecessary(overrides.getHost(), uriParameters.getHost());
```

Comment:
Yes, I'll leave that for another PR since I won't be able to fully remove it from there until I can synchronise with @elrodro83 regarding oauth. Though that might be in the last PR of the series, my next one will add all the payload variables and such to replace it.